src/routes/projects/[id([0-9]+)]/index.svelte

```diff
@@ -65,15 +67,24 @@
       try {
         const application = await api.json(api.post(
           `/projects/${activity.project}/activities/${activity.id}/applications`,
-          { data: { telegram, comment } },
+          { data: { telegram, comment }, csrfToken: account.csrf_token },
         ));
         application.applicant = account;
         activity.existing_application = application;
         project = project;
         projectStore.set(project);
 
         if (telegram && telegram != account.telegram) {
-          api.patch('/account/telegram', { data: { telegram_username: telegram } });
+          try {
+            await api.json(api.patch(
+              '/account/telegram',
+              { data: { telegram_username: telegram }, csrfToken: account.csrf_token },
+            ));
+            account.telegram = telegram;
+            $session.account = account;
```

Comment:
Comment from profile doesn't apply here, but perhaps it is better to use a store method like `.set` or `.update`?

Reply:
Why? It would be more verbose. I understand that it ensures no subscriptions, but then again, perhaps the Svelte compiler is smart enough to recognize this. So far I see no reason for it.